diff --git a/flipendo/flipendo.te b/flipendo/flipendo.te
deleted file mode 100644
index cdcffb2..0000000
--- a/flipendo/flipendo.te
+++ /dev/null
@@ -1,17 +0,0 @@
-type flipendo, domain, coredomain;
-
-app_domain(flipendo)
-
-# Access to Westworld logging service
-allow flipendo fwk_stats_hwservice:hwservice_manager find;
-binder_call(flipendo, stats_service_server)
-
-binder_call(flipendo, gpuservice)
-
-allow flipendo app_api_service:service_manager find;
-
-# Allows Flipendo to change app saturation
-allow flipendo color_display_service:service_manager find;
-
-# Access to PowerHal service
-hal_client_domain(flipendo, hal_power);
diff --git a/flipendo/seapp_contexts b/flipendo/seapp_contexts
deleted file mode 100644
index 773a179..0000000
--- a/flipendo/seapp_contexts
+++ /dev/null
@@ -1,2 +0,0 @@
-# Domain for Flipendo
-user=_app seinfo=platform name=com.google.android.flipendo domain=flipendo type=app_data_file levelFrom=all
\ No newline at end of file
diff --git a/googlebattery/file_contexts b/googlebattery/file_contexts
deleted file mode 100644
index f4fc712..0000000
--- a/googlebattery/file_contexts
+++ /dev/null
@@ -1 +0,0 @@
-/vendor/bin/hw/vendor\.google\.google_battery@1\.0-service-vendor                     u:object_r:hal_googlebattery_exec:s0
diff --git a/googlebattery/hal_googlebattery.te b/googlebattery/hal_googlebattery.te
deleted file mode 100644
index 6e6697b..0000000
--- a/googlebattery/hal_googlebattery.te
+++ /dev/null
@@ -1,13 +0,0 @@
-type hal_googlebattery, domain;
-type hal_googlebattery_exec, exec_type, vendor_file_type, file_type;
-
-init_daemon_domain(hal_googlebattery)
-
-r_dir_file(hal_googlebattery, sysfs_batteryinfo)
-
-allow hal_googlebattery sysfs_batteryinfo:file rw_file_perms;
-allow hal_googlebattery self:netlink_kobject_uevent_socket create_socket_perms_no_ioctl;
-
-hwbinder_use(hal_googlebattery)
-add_hwservice(hal_googlebattery, hal_googlebattery_hwservice)
-get_prop(hal_googlebattery, hwservicemanager_prop)
diff --git a/googlebattery/hwservice.te b/googlebattery/hwservice.te
deleted file mode 100644
index 40323ef..0000000
--- a/googlebattery/hwservice.te
+++ /dev/null
@@ -1 +0,0 @@
-type hal_googlebattery_hwservice, hwservice_manager_type;
diff --git a/googlebattery/hwservice_contexts b/googlebattery/hwservice_contexts
deleted file mode 100644
index 40f1526..0000000
--- a/googlebattery/hwservice_contexts
+++ /dev/null
@@ -1 +0,0 @@
-vendor.google.google_battery::IGoogleBattery                    u:object_r:hal_googlebattery_hwservice:s0
diff --git a/googlebattery/platform_app.te b/googlebattery/platform_app.te
deleted file mode 100644
index 0ee586f..0000000
--- a/googlebattery/platform_app.te
+++ /dev/null
@@ -1,3 +0,0 @@
-# allow SystemUI to find and bind Google Battery HAL
-allow platform_app hal_googlebattery_hwservice:hwservice_manager find;
-binder_call(platform_app, hal_googlebattery)
diff --git a/googlebattery/system_app.te b/googlebattery/system_app.te
deleted file mode 100644
index 05723bf..0000000
--- a/googlebattery/system_app.te
+++ /dev/null
@@ -1,3 +0,0 @@
-# To allow Settings to find and bind Google Battery HAL
-allow system_app hal_googlebattery_hwservice:hwservice_manager find;
-binder_call(system_app, hal_googlebattery)
diff --git a/googlebattery/turbo_adapter.te b/googlebattery/turbo_adapter.te
deleted file mode 100644
index 33f99cd..0000000
--- a/googlebattery/turbo_adapter.te
+++ /dev/null
@@ -1,3 +0,0 @@
-# To find and bind Google Battery HAL
-allow turbo_adapter hal_googlebattery_hwservice:hwservice_manager find;
-binder_call(turbo_adapter, hal_googlebattery)
diff --git a/turbo_adapter/seapp_contexts b/turbo_adapter/seapp_contexts
deleted file mode 100644
index 4f983be..0000000
--- a/turbo_adapter/seapp_contexts
+++ /dev/null
@@ -1 +0,0 @@
-user=_app seinfo=platform name=com.google.android.turboadapter domain=turbo_adapter type=app_data_file levelFrom=all
diff --git a/turbo_adapter/turbo_adapter.te b/turbo_adapter/turbo_adapter.te
deleted file mode 100644
index b2f0202..0000000
--- a/turbo_adapter/turbo_adapter.te
+++ /dev/null
@@ -1,13 +0,0 @@
-# Normal platform_apps cannot access PowerHAL, so we need to define our own domain. Unfortunately
-# this means that TurboAdapter doesn't get the platform_app permissions any more, so we need to
-# list everything that it needs here.
-
-type turbo_adapter, domain, coredomain;
-
-app_domain(turbo_adapter)
-
-# To use ServiceManager
-allow turbo_adapter app_api_service:service_manager find;
-
-# To find and call hal_power_default so turbo can obtain the service extension (IPowerExt)
-hal_client_domain(turbo_adapter, hal_power)
